pages/team-members/index.tsx

```diff
@@ -1,3 +1,4 @@
+import React from "react";
```

Comment:
You don't need to import React into `.tsx` files. You can disable the error by adding ```  rules: {
    "react/react-in-jsx-scope": "off"
  }``` to your `.eslintrc.js`